### ISSUE TITLE
Automatic Import?

### ISSUE THREAD

**Issue by Creative-FlowIT** [thundernest/import-export-tools-ng#36]
Hey, I want to provide a **Thunderbird** e-mail solution for my colleagues.
My idea is that we have a directory on a NAS and each client automatically imports e-mail templates.
It would be sufficient if the automatic import is only executed every time **Thunderbird** is started.
But it would be even better if you could define a time interval by checking if new templates exist (and if so importing them automatically).

Thanks in advance,
TM

**Comment by cleidigh** [comment]
@Creative-FlowIT 
Thanks for the post!  as you probably already know, the extension currently does not do anything with templates.  That might be stretching its mandate.
However, there is an extension by Axel G , SmartTemplate that might do what you want.
@RealRaven2000 care to comment?
https://addons.thunderbird.net/en-US/thunderbird/addon/smarttemplate4/?src=search

**Comment by Creative-FlowIT** [comment]
Please ignore the fact that there are templates, I just need an automatic import of mbox files into the templates folder in **Thunderbird**.

**Comment by cleidigh** [comment]
okay I get that.
I am wondering if the task system could help.  I will take a look.

**Comment by Creative-FlowIT** [comment]
Anything new?

**Comment by cleidigh** [comment]
@Creative-FlowIT 
I was hoping that Thunderbirds Task/Event system might do the trick.  It's really only related to calendar tasks not executing functions.  What just came to me is another extension I have used for years, AwesomeAutoArchive.
https://addons.thunderbird.net/en-US/thunderbird/addon/awesome-auto-archive/?src=search
It's geared toward scheduled archiving, but it also does moving and  Copying.  in fact that's exactly what I do for my wife's office, we use it to transfer old email to a NAS.
There may also be other extensions but I do not know anything specific.

I do think incorporating a scheduler into IET NG would be a nontrivial task.  But I don't think I can do that at this point given I'm still trying to keep up with making sure the existing functionality works with all Thunderbirds changes.  Take a look at the other extension and let me know.  One issue is that the author still needs to go the last steps to get it upgraded for 68.  It has some activity from last month so I'm hopeful that will happen.  Otherwise it may be something we get someone to adopt as the Thunderbird team (including myself as extension liaison) are working hard to get all the top extensions updated, not necessarily easy!

**Comment by Creative-FlowIT** [comment]
Thanks for the reply.
As I already said, it would be sufficient if Thunderbird imported templates when starting.
It would be great if at least that would be added. The scheduling is something for the future.

Greetings from Germany,
TM

**Comment by cleidigh** [comment]
@Creative-FlowIT 
I agree doing something like this at startup reduces the scheduling however it still got a lot of configuration aspects. With all the new Thunderbird changes I cannot currently add  new features
until everything else in terms of maintenance is taking care of.
That's why I highly recommend looking at the archiver, along with another local folder I believe you should be able to do exactly what you want to do including scheduling.
@cleidigh
